tests/integration_test/ota_pal/test_files/rsa-sha256-signer.key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Why are we adding private keys here ? We should remove this and have instructions on how to generate.